src/link/object.c

```diff
@@ -331,6 +334,29 @@ static void linkSymToSect(struct Symbol const *symbol, struct Section *section)
 	section->nbSymbols++;
 }
 
+/**
+ * Reads an assertion from a file
+ * @param file The file to read from
+ * @param assert The struct to fill
+ * @param fileName The filename to report in errors
+ */
+static void readAssertion(FILE *file, struct Assertion *assert,
+			  char const *fileName, struct Section *fileSections[],
+			  uint32_t i)
+{
+	static char assertName[sizeof("Assertion #XXX")];
```

Comment:
Should be `sizeof("Assertion #") + 10`, since that's the maximum length of an unsigned value.

Reply:
I went with `sizeof("Assertion #" EXPAND_AND_STR(UINT32_MAX))`, which yields the same thing. Good catch, though!